sdk/communication/azure-communication-networktraversal/azure/communication/networktraversal/_shared/user_credential.py

```diff
@@ -79,12 +79,8 @@ def _wait_till_inprogress_thread_finish_refreshing(self):
         self._lock.acquire()
 
     def _token_expiring(self):
-        return self._token.expires_on - self._get_utc_now() <\
-            timedelta(minutes=self._ON_DEMAND_REFRESHING_INTERVAL_MINUTES)
+        return self._token.expires_on - get_current_utc_as_int() <\
+            timedelta(minutes=self._ON_DEMAND_REFRESHING_INTERVAL_MINUTES).total_seconds()
```

Comment:
Why is it now the .total_seconds() added?

Reply:
Because the `self._token.expires_on` is an int (as you can see in utils.py), the  `get_current_utc_as_int()` is an int, and the difference is therefore also an int. If you'd compare it with a timedelta you'd get:
> TypeError: '<' not supported between instances of 'int' and 'datetime.timedelta'

I don't think this has ever worked. The tests were disabled. We enabled them and added new ones in the follow-up PR: https://github.com/Azure/azure-sdk-for-python/pull/21834/files